### PR TITLE
Add training-exempt tag for alphafold and interactive tools

### PR DESCRIPTION
**Pull request by cat-bro**
Add a tag to prevent tools unsuitable for training destinations from being sent there. Instead the destination is unchanged regardless of whether the user is part of a training group.

### CODE REVIEW

**Comment by cat-bro** [comment]
@nuwang if you approve this PR could you please not merge it? I will merge it at a time when I am able to test it.

**Review by nuwang — COMMENTED** [review]
These rules should probably be named as well.
And I wonder whether this matcher could be moved to a helper function, either Galaxy Australia specific, or if it's general enough, into TPV itself, since it has been used several times.

File: files/galaxy/dynamic_job_rules/production/total_perspective_vortex/roles.yml
Comment (cat-bro) [reply files/galaxy/dynamic_job_rules/production/total_perspective_vortex/roles.yml:12]: Hi @nuwang thanks for this.  We are on version 1.x until we switch to v2. I'll add your suggested change to the TPV v2 PR which is currently in draft state.

**Review by nuwang — APPROVED** [review]

File: files/galaxy/dynamic_job_rules/production/total_perspective_vortex/roles.yml
Comment (cat-bro) [reply files/galaxy/dynamic_job_rules/production/total_perspective_vortex/roles.yml:12]: `mapper.entities["tools"]` seems to be every tool in tools.yml

**Pull request merged**
